### PR TITLE
catalog_hovers

### PR DESCRIPTION
**Pull request by SerhiiShapovalenko**
- [DEMO LINK](https://SergeyShapovalenko.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition-duration: $transmission-duration;
```
Comment (loralevitska) [inline src/styles/blocks/header.scss:42]: Add property to  transition image
Fix everywhere

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost perfect! Let's improve

File: src/styles/variables/variables.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+$transition: transform 300ms, color 300ms;
```
Comment (slavanosatskyi) [inline src/styles/variables/variables.scss:1]: minor in real life such variables are rear cases. in your case it was better to move 300ms to variable only. As nav links do not change their transform property on hover event.

**Review by slavanosatskyi — APPROVED** [review]
Nice work!

**Pull request closed without merging**